src/contracts/xMETRIC.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.13;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
-contract xMETRIC is ERC20("xMETRIC", "xMETRIC"), Ownable {
+contract XmETRIC is ERC20("xMETRIC", "xMETRIC"), ERC20Pausable, Ownable {
```

Comment:
Our Linter and I believe it's a common pattern to always name your contract with an Uppercase